test/wadler_test.rb

```diff
@@ -5,7 +5,7 @@
 require_relative 'lib'
 
 def check_roundtrip(margin, expected, builder_block)
-  printer = Oppen::Wadler.new(margin:)
+  printer = Oppen::Wadler.new(margin:, config: nil)
```

Comment:
Why are we sending a `config: nil`? We shouldn't be sending anything and just use the default, no?

Reply:
If added it just after meeting for you to comment on the pr. The tests do not have the desired output with the Confog.wadler. I was looking into it just now